src/content/app-platform/defaulting-validation/index.md

```diff
@@ -109,6 +109,11 @@ Currently we validate:
 - All referenced configmaps and secrets exist.
 - The catalog has a matching AppCatalog CR.
 
+If app-operator finds a matching AppCatalogEntry CR named `{appCatalog}-{app}-{version}`, it will run more two validation.
```

Comment:
```suggestion
If app-operator finds a matching [AppCatalogEntry]({{< relref "/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}}) CR, it will use this to run more validation checks.
```

I don't think we need to show the naming convention but we could link to the CRD docs. I just removed the two as we may add more checks in the future.

Reply:
Nice that we put the link on the CRD document. Thanks!